### PR TITLE
chore: bumps and prettier

### PR DESCRIPTION
**Pull request by FauxFaux**
* upgrade all the dev deps to appropriate versions
   * fix test issues, hanging (new sequelize) and mocha option name changes
 * add an `env` var to facilitate running the tests locally
 * run `prettier` on the codebase with our standard settings

### CODE REVIEW

File: test/index.js [hunk 1]
```diff
@@ -2,7 +2,8 @@ import assert from 'assert';
 import Sequelize from 'sequelize';
 import EncryptedField from '../';
 
-const sequelize = new Sequelize('postgres://postgres@db:5432/postgres');
+const dbHost = process.env.DB_HOST || 'db';
+const sequelize = new Sequelize(`postgres://postgres@${dbHost}:5432/postgres`);
```
Comment (michael-go) [inline test/index.js:6]: would reword the commit title to indicate it's override in the tests

File: .prettierrc.json [hunk 2]
```diff
@@ -0,0 +1,6 @@
+{
```
Comment (michael-go) [inline .prettierrc.json:1]: hmm ... this will make it hard to merge it back to upstream if we actually want to do this one day

Comment (FauxFaux) [reply .prettierrc.json:1]: It would. It is very abandoned upstream, though. No commits since 2016, open issues/PRs with no feedback since 2017.
